benchmarks/torchbench/benchfile.py

```diff
@@ -1,13 +1,39 @@
+import contextlib
+import os
+import subprocess
 from milabench.pack import Package
 
 BRANCH = "ff7114655294aa3ba57127a260dbd1ef5190f610"
 
 
+@contextlib.contextmanager
+def tmp_path(path):
+    curr_path = os.environ["PATH"]
+    os.environ["PATH"] = path
+
+    yield
+
+    os.environ["PATH"] = curr_path
+
+
 class TorchBenchmarkPack(Package):
     def install(self):
         code = self.dirs.code
-        code.clone_subtree("https://github.com/pytorch/benchmark", BRANCH)
+        data = self.dirs.data
+        env_path = os.environ["PATH"]
+        try:
+            subprocess.check_call(["which", "git-lfs"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        except subprocess.CalledProcessError as e:
+            subprocess.check_call([code / "download_extract_git-lfs.sh"], cwd=data, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+            self._nox_session.env["PATH"] += f":{data / 'bin'}"
+            env_path += f":{data / 'bin'}"
+        with tmp_path(env_path):
+            code.clone_subtree("https://github.com/pytorch/benchmark", BRANCH)
+        os.makedirs(code / ".git/branches", exist_ok=True)
+        os.makedirs(code / ".git/refs", exist_ok=True)
         self.pip_install("-r", code / "requirements-bench.txt")
+        if os.path.exists(code / f"requirements-{self.config['model']}.txt"):
+            self.pip_install("-r", code / f"requirements-{self.config['model']}.txt")
         self.python("install.py", "--models", self.config["model"])
```

Comment:
Was the `install.py` invocation insufficient for these models?

Reply:
Not always. It didn't installed some required dependencies for some models